tests/unit/sagemaker/serve/mode/test_in_process_mode.py

```diff
@@ -0,0 +1,152 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import unittest
+from unittest.mock import patch, Mock
+
+from sagemaker.serve.mode.in_process_mode import InProcessMode
+from sagemaker.serve import SchemaBuilder
+from sagemaker.serve.utils.types import ModelServer
+from sagemaker.serve.utils.exceptions import LocalDeepPingException
+
+
+mock_prompt = "Hello, I'm a language model,"
+mock_response = "Hello, I'm a language model, and I'm here to help you with your English."
+mock_sample_input = {"inputs": mock_prompt, "parameters": {}}
+mock_sample_output = [{"generated_text": mock_response}]
+
+
+class TestInProcessMode(unittest.TestCase):
```

Comment:
I assume this will be removed on revision

Reply:
Yes, I will be removing in_process mode changes as that is a different PR: https://github.com/aws/sagemaker-python-sdk/pull/4784